src/main/lib/utils.js

```diff
@@ -1,14 +1,15 @@
 import path from 'path';
 
-export function getPathToFile(fileName, disableFilePath = false) {
+export function getPathToFile(fileName) {
   if (process.env.NODE_ENV === 'development') {
-    return `${disableFilePath ? '/' : 'file://'}${path.resolve(
-      __dirname,
-      `../../../app/${fileName}`
-    )}`;
+    return `file://${path.resolve(__dirname, `../../../app/${fileName}`)}`;
```

Comment:
Sorry to be pedantic! But `getPathToFile` should return something like `/path/to/file` and `getURIPathToFile` should return the `file:///path/to/file` URL.

To make it simpler, could could make `getPathToFile`, and then:

```javascript
function getURIPathToFile(fileName) {
  return `file://${getPathToFile(fileName)}`;
}
```

(Most likely)

Reply:
Sure :)